src/lib/validations/sql.js

```diff
@@ -48,6 +48,29 @@ const infoCheckFormatter = check => {
 	} );
 };
 
+function checkTablePrefixes( tables ) {
```

Comment:
I moved this code up to be closer to where it is called.  I found it useful to move it when reading the file.